gef.py

```diff
@@ -6938,7 +6921,6 @@ def update_registers(cls, event):
 
     def empty_extra_messages(self, event):
         global __context_messages__
-        del __context_messages__
         __context_messages__ = []
```

Comment:
So because you were deleting `__context_message__` here, this was actually just creating a local version (it seems)